src/_components/SideMenuComponent.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import { Icon, Menu, Segment, Sidebar } from 'semantic-ui-react';
+
+import NavBarComponent from './NavBarComponent';
+
+import '../_css/SideMenuComponent.css';
+
+class SideMenuComponent extends Component {
+  state = { visible: false };
+
+  toggleVisibility = () => this.setState({ visible: !this.state.visible });
+
+  render() {
+    const { visible } = this.state;
+    const { title } = this.props;
+
+    return (
+      <div className='wrapper'>
+        <Sidebar.Pushable as={ Segment }>
```

Comment:
Only destructured objects  should have the space after `{` or before `}`. Values passed to elements shouldn't have the spaces. So this should just be `as={Segment}`. I'll talk to Eric about integrating prettier to the codebase so as not to worry about the nitty-gritty

Reply:
Awesome. Let me rectify that.